index.js

```diff
@@ -0,0 +1,162 @@
+const teachersDictionary = [
+  {
+    title: 'SehYeong Noh',
+    description: 'Director of Frontend department and member of University board.',
+    descriptionn: 'SehYeong Noh has attended his Masters and Phd degree in this university and He was amongest the best achiveres and he is currently Teaching in this university.',
+    img: './img/teach.png',
+    alt: 'teacher',
+  },
+  {
+    title: 'Ryan Merkley',
+    description: 'Director of Frontend department and member of University board.',
+    descriptionn: 'SehYeong Noh has attended his Masters and Phd degree in this university and He was amongest the best achiveres and he is currently Teaching in this university.',
+    img: './img/teach1.png',
+    alt: 'teacher',
+  },
+  {
+    title: 'Julia Leda',
+    description: 'Director of Frontend department and member of University board.',
+    descriptionn: 'SehYeong Noh has attended his Masters and Phd degree in this university and He was amongest the best achiveres and he is currently Teaching in this university.',
+    img: './img/teach2.png',
+    alt: 'teacher',
+  },
+  {
+    title: 'Kilnam Chon',
+    description: 'Director of Frontend department and member of University board.',
+    descriptionn: 'SehYeong Noh has attended his Masters and Phd degree in this university and He was amongest the best achiveres and he is currently Teaching in this university.',
```

Comment:
Your images seem to have a broken link please kindly fix this issue. AAND PLEASE  FOLLOW. The given design.





![Screenshot 2023-04-28 at 9 53 09 PM](https://user-images.githubusercontent.com/79506105/235249708-74d3d959-0d22-4d6b-b85a-18720abc4683.png)
![Screenshot 2023-04-28 at 9 53 00 PM](https://user-images.githubusercontent.com/79506105/235249724-d95c89b8-5bae-4de2-bb9a-9d90b0782d3d.png)